docs/reference/collection-configuration.txt

```diff
@@ -10,51 +10,88 @@ Collection Configuration
    :depth: 2
    :class: singlecol
 
+Configuring a Document Collection
+=================================
 
-Capped Collections
-------------------
+You can specify collection options for documents using ``store_in`` macro.
+This macro accepts ``:collection_options`` argument, which can contain any collection
+options that is supported by the driver.
+
+.. note::
+
+  In order to apply the options, the collection must be explicitly created up-front.
+  This should be done using  :ref:`Collection Management Rake Task<collection-management-task>`.
+
+Please refer to `the driver collections page
+<https://mongodb.com/docs/ruby-driver/current/reference/collection-tasks/>`_
+for the more information about collection options.
```

Comment:
For indexes there is a documented way to create them by directly executing the respective method without going through a rake task: https://www.mongodb.com/docs/mongoid/master/reference/indexes/#index-management-rake-tasks

I use this all the time and I think this pattern should be continued for the collection creation as well.

Reply:
Done